community/values/src/main/java/org/neo4j/values/result/QueryResult.java

```diff
@@ -17,18 +17,38 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.bolt.v1.messaging.infrastructure;
+package org.neo4j.values.result;
```

Comment:
Yes I agree. It is a little bit values related since the visitor uses values. My grand plan was to have the result being defined outside cypher but maybe it should live in its own module. I don't know, lets discuss and move in a separate PR?